wicket-core/src/test/java/org/apache/wicket/markup/MarkupParserTest.java

```diff
@@ -67,31 +67,31 @@ void tagParsing() throws Exception
 		final ComponentTag aOpen = (ComponentTag)markupStream.next();
 
 		log.info("", aOpen);
-		assertTrue(aOpen.getName().equals("a"));
+		assertEquals(aOpen.getName(), "a");
 		assertEquals("foo.html", aOpen.getAttributes().getString("href"));
 
 		markupStream.next();
 
 		final ComponentTag boldOpen = (ComponentTag)markupStream.next();
 
 		log.info("", boldOpen);
-		assertTrue(boldOpen.getName().equals("b"));
+		assertEquals(boldOpen.getName(), "b");
```

Comment:
Please do another change:

`assertEquals` defined as follows: `assertEquals(expected, actual)` so these 2 arguments need to be swapped

Please ensure the order is correct for all asserts you have changed :)